packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -110,6 +110,19 @@ const isLimitExceeded = async (orgId: string, dataLoader: DataLoaderWorker) => {
     .run()
 }
 
+export const removeTeamLimitsJobs = async (orgId: string) => {
+  const removeJobTypes = ['LOCK_ORGANIZATION', 'WARN_ORGANIZATION']
+  return r
+    .table('ScheduledJob')
+    .filter((row: RValue) => {
```

Comment:
-1 I'm a bit worried about performance here. Right now we're not having many `ScheduledJob`s, but this will change once we enable the team limit. For this case we should make better use of indexes. I think creating one for `orgId` should be sufficient, so we can do
```ts
r
    .table('ScheduledJob')
    .getAll(orgId, {index: 'orgId'})
    .filter((row: RValue) => (r.expr(removeJobTypes).contains(row('type')))_
    .delete()
    .run()
```